hack/README.md

```diff
@@ -0,0 +1,34 @@
+# Standard build environment
+
+This is a set of tools for constructing a standard Go build environment for OpenShift.
+
+1. To start, copy the `tools/hack/golang` directory into the root of your repo as `hack`.
+2. Move `package.spec` into your root repository directory as `<REPO_NAME>.spec` (this will be your RPM build)
+3. Move `Makefile` into your root repository directory
```

Comment:
why is this not just how the repo comes? i'm confused by this.

Reply:
This file is out of date.